CRM/Queue/BAO/Queue.php

```diff
@@ -18,7 +18,7 @@
 /**
  * Track a list of known queues.
  */
-class CRM_Queue_BAO_Queue extends CRM_Queue_DAO_Queue {
+class CRM_Queue_BAO_Queue extends CRM_Queue_DAO_Queue implements \Civi\Test\HookInterface {
```

Comment:
I think this is now \Civi\Core\HookInterface

Reply:
@eileenmcnaughton Yup, that's prettier. Pushed update.